rudder-server/SOURCES/patches/jetty/jetty-init-prevent-false-failed-starts.patch

```diff
@@ -1,20 +1,6 @@
---- jetty/bin/jetty.sh	2017-10-20 21:09:54.213736038 +0200
-+++ jetty/bin/jetty.sh_5	2017-10-20 21:18:07.960397445 +0200
-@@ -137,11 +137,11 @@ started()
-   do
-     sleep 4
-     [ -z "$(tail -1 $1 | grep STARTED 2>/dev/null)" ] || return 0
-+
-     [ -z "$(tail -1 $1 | grep STOPPED 2>/dev/null)" ] || return 1
-     [ -z "$(tail -1 $1 | grep FAILED 2>/dev/null)" ] || return 1
--    local PID=$(cat "$2" 2>/dev/null) || return 1
--    kill -0 "$PID" 2>/dev/null || return 1
-     echo -n ". "
```

Comment:
this one, I *really* don't know what it became. Things seems to have  change a lot in the jetty start script. And our git history is broken because of rudder-server new package. I'm not sure what to do with that.